### DISCUSSION TITLE
Can the camera go below "sea level"

### DISCUSSION THREAD

**Discussion by frobones** [heremaps/harp.gl#2043] [Q&A] [answered]
Hi all,

For my project, I need to render 3D objects below the surface of the map (I guess this is sea level), which works. However, I would also like the camera to be able to rotate enough to be able to view the underside of the 3D object. It looks like the camera tilt is locked between 0 and 89 degrees: https://www.harp.gl/docs/master/doc/classes/harp_map_controls.mapcontrols-1.html#maxtiltangle

I tried modifying `MapControls.js` (https://github.com/heremaps/harp.gl/blob/44b5a42/@here/harp-map-controls/lib/MapControls.ts#L59) to remove this clamp but when the camera goes below the map surface, nothing is rendered.

So my question, is it possible for the system to have the camera view the map from below the surface before I spend time trying to figure out how to accomplish this?

**Comment by nzjony** [comment, marked as answer]
@frobones Thanks for your question.

Check out the example: https://www.harp.gl/docs/master/examples/#getting-started_free-camera.html it shows how you can setup a second camera which lets you look below the map.

We have had other teams request such a feature which would not require the second camera, but we haven't yet implemented it. However, if it doesn't work so nicely with the second camera as I mentioned above, maybe we can look to work on this soon.

**Reply by frobones** [reply]
Awesome, this is great. Thank you.
